### PR TITLE
Genesis params

### PR DESCRIPTION
**Pull request by faza**

### CODE REVIEW

File: x/gitopia/keeper/genesis.go [hunk 1]
```diff
@@ -151,9 +152,17 @@ func InitGenesis(ctx sdk.Context, k Keeper, genState types.GenesisState) {
...
 
-		if err := k.createModuleAccount(ctx, types.LiquidityBootstrappingPoolAccountName, totalLiquidityBootstrappingPoolCoins); err != nil {
+		if err := k.createModuleAccount(ctx, types.LiquidityBootstrappingPoolAccountName, coin); err != nil {
+			panic(err)
+		}
+
+		addr := k.accountKeeper.GetModuleAddress(types.LiquidityBootstrappingPoolAccountName)
+		balances := k.bankKeeper.GetAllBalances(ctx, addr)
+
+		err := k.distrKeeper.FundCommunityPool(ctx, balances, addr)
```
Comment (ProgramCpp) [inline x/gitopia/keeper/genesis.go:164]: why do we move liquidity pool balance to community pool?

Comment (faza) [reply x/gitopia/keeper/genesis.go:164]: @ProgramCpp We wanted the ability to move lbp funds via governance. Moving it into community pool enables us to do it via community spend proposal

File: cmd/gitopiad/cmd/genesis.go [hunk 2]
```diff
@@ -46,24 +49,59 @@ const (
...
-	strategicPartnersMultiSigAddress = ""
-	advisorsMultiSigAddress          = ""
-	teamMultiSigAddress              = ""
+	rewardsServiceAddress            = "gitopia1r7mhw4f6x73lap55st3yxn4u7tj4yt6qqwmlv5"
+	strategicReserveAddress1         = "gitopia1ryv3840cdfjhknqrjkvuhyn6h4zylwvq9ekdv9"
+	strategicReserveAddress2         = "gitopia12lehgzv7w7k8x0vwu9nsu9fvj9e80evhsu3jpf"
+	strategicReserveAddress3         = "gitopia17505m7wwzh28sf6dud2a8rd8qztlp4xvz5nqca"
+	strategicReserveAddress4         = "gitopia17urengx7kxrsche82cfxu3qutxnng97k04vy9a"
+	strategicReserveAddress5         = "gitopia1hcq6ejnxmsxaqpee7vh7mqjysgw2kp8sf4g4xm"
+	feegrantsAddress                 = "gitopia13ashgc6j5xle4m47kqyn5psavq0u3klmscfxql"
```
Comment (ProgramCpp) [inline cmd/gitopiad/cmd/genesis.go:58]: where does the fee grants balance come from?

File: cmd/gitopiad/cmd/genesis.go [hunk 3]
```diff
@@ -798,6 +979,14 @@ func GenerateGenesisCmd() *cobra.Command {
...
 			)
 
+			slashingGenesis.Params = slashingtypes.NewParams(
+				50000,
+				sdk.MustNewDecFromStr("0.05"),
+				60*10*time.Second,
+				sdk.MustNewDecFromStr("0.05"),
+				sdk.MustNewDecFromStr("0.0001"),
+			)
+
```
Comment (ProgramCpp) [inline cmd/gitopiad/cmd/genesis.go:989]: the reward pool does not equal to 60 mil. how do we mint 60 mil for airdrop?

File: cmd/gitopiad/cmd/genesis.go [hunk 4]
```diff
@@ -46,24 +49,59 @@ const (
...
-	advisorsMultiSigAddress          = ""
-	teamMultiSigAddress              = ""
+	rewardsServiceAddress            = "gitopia1r7mhw4f6x73lap55st3yxn4u7tj4yt6qqwmlv5"
+	strategicReserveAddress1         = "gitopia1ryv3840cdfjhknqrjkvuhyn6h4zylwvq9ekdv9"
+	strategicReserveAddress2         = "gitopia12lehgzv7w7k8x0vwu9nsu9fvj9e80evhsu3jpf"
+	strategicReserveAddress3         = "gitopia17505m7wwzh28sf6dud2a8rd8qztlp4xvz5nqca"
+	strategicReserveAddress4         = "gitopia17urengx7kxrsche82cfxu3qutxnng97k04vy9a"
+	strategicReserveAddress5         = "gitopia1hcq6ejnxmsxaqpee7vh7mqjysgw2kp8sf4g4xm"
+	feegrantsAddress                 = "gitopia13ashgc6j5xle4m47kqyn5psavq0u3klmscfxql"
+	earlySupportersMultiSigAddress   = "gitopia1l8d76cnr6z5f02unp3hqayccqg789d9u0knxce"
```
Comment (ProgramCpp) [inline cmd/gitopiad/cmd/genesis.go:59]: which pool does early supporters receive funds from?

File: cmd/gitopiad/cmd/genesis.go (hunk 2 above)
Comment (faza) [reply cmd/gitopiad/cmd/genesis.go:58]: https://github.com/gitopia/gitopia/blob/785e5ae1c569f2437d848871aac92b78c7f0ee2b/cmd/gitopiad/cmd/genesis.go#L870

File: cmd/gitopiad/cmd/genesis.go [hunk 5]
```diff
@@ -46,24 +49,59 @@ const (
...
+	addr, _ := sdk.AccAddressFromBech32(address)
+	baseAccount := authtypes.NewBaseAccountWithAddress(addr)
+
+	var periods []authvesting.Period
+
+	vestedTokens := math.NewInt(tokens).Mul(math.NewInt(5)).Quo(math.NewInt(100))
+
+	for i := 0; i < 20; i++ {
+		periods = append(periods, authvesting.Period{
+			Length: int64(VESTING_PERIOD),
```
Comment (ProgramCpp) [inline cmd/gitopiad/cmd/genesis.go]: the length is expected to be in seconds but the vesting period is in no of blocks unit.

**Review by hTrap — APPROVED** [review]

**Pull request merged**